pyproject.toml

```
@@ -41,47 +42,31 @@ test = [
     "pytest-cov",
     "pytest>=6.0",
 ]
-
 [tool.hatch.version]
 source = "nodejs"
 
-[tool.hatch.build.hooks.version]
-path = "ipylgbst/_version.py"
-
 [tool.hatch.metadata.hooks.nodejs]
-fields = ["description", "authors", "urls"]
+fields = ["description", "authors", "urls", "keywords"]
 
-[tool.hatch.build]
-artifacts = [
-    "ipylgbst/nbextension/index.*",
-    "ipylgbst/labextension/*.tgz",
-    "ipylgbst/labextension",
-]
+[tool.hatch.build.targets.sdist]
+artifacts = ["ipylgbst/labextension"]
+exclude = [".github", "binder"]
 
 [tool.hatch.build.targets.wheel.shared-data]
-"ipylgbst/nbextension" = "share/jupyter/nbextensions/ipylgbst"
 "ipylgbst/labextension" = "share/jupyter/labextensions/ipylgbst"
-"./install.json" = "share/jupyter/labextensions/ipylgbst/install.json"
-"./ipylgbst.json" = "etc/jupyter/nbconfig/notebook.d/ipylgbst.json"
+"install.json" = "share/jupyter/labextensions/ipylgbst/install.json"
```

```suggestion
"install.json" = "share/jupyter/labextensions/ipylgbst/install.json"
"ipylgbst.json" = "etc/jupyter/nbconfig/notebook.d/ipylgbst.json"
```
Same idea as before.